### PR TITLE
Updating email partials on the site

### PR DESCRIPTION
**Pull request by alexandergimson**
This commit updates the email partials that were on the site before. We found that segments in customer.io weren't being created properly. I've now made new segments in customer.io with new emails. This makes sure that the email provided goes into the correct segment

@nikeshashar

### CODE REVIEW

File: button.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<p style="margin: 18pt 0px; padding: 0px; color: rgb(48, 55, 61); font-size: 18px; line-height: 1.6; text-align: center;"><a target="_blank" href="https://docs.google.com/uc?authuser=0&amp;id=0B6qWOHggTmDLSW1VbUxYNkU1X1k&amp;export=download" style="margin: 0px; padding: 14px; color: rgb(170, 25, 37); text-decoration: none; border: 2px solid rgb(6, 37, 51); opacity: 0.9; -webkit-font-smoothing: antialiased; border-radius: 8px; cursor: pointer; display: inline-block; line-height: 27px; min-width: 5em; vertical-align: middle; white-space: nowrap; background-color: transparent;" class="button">Download your e-book now!</a></p>
```
Comment (nikeshashar) [inline button.html]: what happened here?

Comment (alexandergimson) [inline button.html]: I haven't got a clue what this is about dude. I literally have never made this!

File: source/entrepreneurs.html.haml [hunk 2]
```diff
@@ -22,7 +22,7 @@
       %h3 This could be you
       %p Our graduates have gone on to build successful businesses, been accepted into accelerators, and secured further funding. Read their stories below.
 
-= partial :"full_graduates", locals: { category: "entrepreneur" }
+    = partial :"email-capture", locals: { segments: "website-entrepreneurs" }
```
Comment (nikeshashar) [inline source/entrepreneurs.html.haml]: have you changed this from a partial of the graduates to an email capture?

Comment (alexandergimson) [inline source/entrepreneurs.html.haml]: Shoot. Good spot, I'll change it ASAP. Sorry.

File: source/learn-ruby-in-15.html.haml [hunk 3]
```diff
@@ -54,7 +54,7 @@
     .centered-row
       %h1 Learn Ruby In 15 Minutes
       %h4.subheader Download the guide
-    = partial :"email-capture"
+    = partial :"email-capture", locals: { segments: "website-entrepreneurs" }
```
Comment (nikeshashar) [inline source/learn-ruby-in-15.html.haml]: this is the same page as the partial above? it should have the same segment for each capture.

File: button.html (hunk 1 above)
Comment (nikeshashar) [inline button.html]: Checking this with @sjmog

Comment (sjmog) [inline button.html]: Huh, this is new in this PR so it must have been hanging about on your computer for some reason @alexandergimson . Wild. If you're not using it, kill it!

File: source/entrepreneurs.html.haml [hunk 4]
```diff
@@ -22,7 +22,7 @@
       %h3 This could be you
       %p Our graduates have gone on to build successful businesses, been accepted into accelerators, and secured further funding. Read their stories below.
 
-= partial :"full_graduates", locals: { category: "entrepreneur" }
+    = partial :"full_graduates", locals: { category: "entrepreneur" }
```
Comment (sjmog) [inline source/entrepreneurs.html.haml]: Could you super-check that sticking this partial inside an `article--centered` doesn't break things on mobile or on desktop?

**Comment by nikeshashar** [comment]
@sjmog Alex isn't here for a week. Could you take a final look on mobile. I think I've fixed the little errors

**Comment by nikeshashar** [comment]
:shipit:

**Pull request merged**